dqc_us_rules/dqc_us_0041.py

```diff
@@ -288,17 +304,19 @@ def _catch_dqc_us_0041_errors(val):
         XbrlConst.dimensionDefault
     ).modelRelationships
     for rel in rel_set:
-        rel_to = rel.toModelObject
-        rel_from = rel.fromModelObject
-        if _default_dimension_mismatch(
-            rel_to.name,
-            val.usgaapDefaultDimensions[rel_from.name]
-        ):
-            yield (
-                rel_from.name,
-                val.usgaapDefaultDimensions[rel_from.name],
-                rel_to.name
-            )
+        if _check_relationship_exists(rel):
```

Comment:
CR: @andrewperkins-wf should nothing be thrown if the relationship does not exist, or does that still violate the spirit of the rule?